cmr-exchange/metadata-proxy/project.clj

```diff
@@ -15,7 +15,7 @@
        ns
        "\u001B[35m]\u001B[33m λ\u001B[m=> "))
 
-(defproject gov.nasa.earthdata/cmr-metadata-proxy "0.2.1-SNAPSHOT"
+(defproject gov.nasa.earthdata/cmr-metadata-proxy "0.2.2-SNAPSHOT"
```

Comment:
Have you published each of the updated library versions to clojars?

Reply:
Daniel will help me publishing them. Thanks @daniel-zamora